packages/datagateway-search/src/search/searchButton.component.tsx

```diff
@@ -5,6 +5,18 @@ import Button from '@material-ui/core/Button';
 import axios from 'axios';
 import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
 import { format } from 'date-fns';
+// import {
```

Comment:
I presume this is like a WIP for your next issue - but can we remove these?

Reply:
done!